### PR TITLE
Update for Bonita BPM 7.4. 

### PR DESCRIPTION
**Pull request by amottier**
Main change is related to the usaged of setup tools to pull and push REST API extension permissions (configuration) changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,8 +82,15 @@ You can either retrieve the binaries from the [releases](../../releases) and unz
 
 ### Configure REST API authorization
 
-- Edit `custom-permissions-mapping.properties` file located in `<BONITA_HOME>/client/tenants/1/conf/` directory.
+- Only if you are deploying the REST API extension in the Tomcat embedded in Bonita BPM Studio:
+  - Go in `workspace/tomcat/setup`
```
Comment (lio-p) [inline README.md]: Go in `<Bonita_Studio_Path>/workspace/tomcat/setup`

File: README.md [hunk 2]
```diff
@@ -77,8 +82,15 @@ You can either retrieve the binaries from the [releases](../../releases) and unz
...
+  - Edit `database.properties` file
+  - Set the `h2.database` property with the full path to the `h2_database` folder located in `workspace/default` folder (replace the existing `../h2_database` default value).
```
Comment (lio-p) [inline README.md]: `<Bonita_Studio_Path>/workspace/default`

File: README.md [hunk 3]
```diff
@@ -108,20 +120,21 @@ These instructions apply to a standard Tomcat H2 bundle.
...
 ### Deploying the resources in Bonita BPM Portal
 
-- Log in with a user with Administrator profile in Bonita BPM Portal
-- Import the REST API Extension file: `rest-api-sql-datasource.zip`
+- Log in with a user with Administrator profile in Bonita BPM Portal.
+- Switch to administrator view.
+- Go to "Resources" and click on the "Add" button to import the REST API Extension file: `rest-api-sql-datasource.zip`.
 - Import the sample viewer page `page-apiExtensionDatasourceViewer.zip`
-- Import the living application `application.xml`
+- Go to "Applications" and import the living application `application.xml`
```
Comment (lio-p) [inline README.md]: Living

**Review by lio-p — APPROVED** [review]

**Comment by amottier** [comment]
Should be ok now for merge. @lio-p can you do a final review and merge?

**Comment by lio-p** [comment]
Ok for me.

**Pull request merged**